### PR TITLE
SSH persistent tmux config

### PR DESCRIPTION
**Pull request by ddemidov**
This is basically #1689 with the updated changelog, as requested by @pvizeli.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @ddemidov, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;addons#2191).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: ssh/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 9.2.1
```
Comment (ludeeus) [inline ssh/CHANGELOG.md:3]:
```suggestion
## 9.3.0
```

File: ssh/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+
+- Make tmux configuration persistent
```
Comment (ludeeus) [inline ssh/CHANGELOG.md:5]:
```suggestion
- Persist tmux configuration 
```

**Review by ludeeus — CHANGES_REQUESTED** [review]
This needs a version bump in config.json

**Comment by ddemidov** [comment]
> This needs a version bump in config.json

done (714ca80a0e8a630fccb45d9008ff843da35bcabc)

File: ssh/rootfs/etc/cont-init.d/profile.sh [hunk 3]
```diff
@@ -10,6 +10,14 @@ if ! bashio::fs.file_exists /data/.bash_profile; then
     chmod 600 /data/.bash_history
 fi
 
+# Persist tmux configuration by redirecting .tmux.conf to /data
```
Comment (ludeeus) [inline ssh/rootfs/etc/cont-init.d/profile.sh:13]: Thinking about this change a bit more...
If we add this, we can no longer adjust the default configuration for existing installations.
Which will result in new installations behaving differently from existing ones if we need to do that.

@pvizeli what do you think?

Comment (ddemidov) [reply ssh/rootfs/etc/cont-init.d/profile.sh:13]: The reason I picked up the PR is that I did not like the default tmux configuration, and there was no way to update it in a way that would survive a reboot. I guess another option is to put the contents of `tmux.conf` into the addon config?

Comment (ludeeus) [reply ssh/rootfs/etc/cont-init.d/profile.sh:13]: I think it's best to wait with anything like this until <https://github.com/home-assistant/supervisor/issues/1842> is implemented.

Comment (ddemidov) [reply ssh/rootfs/etc/cont-init.d/profile.sh:13]: That indeed looks promising. Should I close this?

**Pull request closed without merging**